packages/models/src/tenant/protobufConverterToV2.ts

```diff
@@ -0,0 +1,151 @@
+import { match } from "ts-pattern";
+import {
+  TenantAttributeV2,
+  TenantFeatureV2,
+  TenantKindV2,
+  TenantMailKindV2,
+  TenantMailV2,
+  TenantRevokerV2,
+  TenantUnitTypeV2,
+  TenantV2,
+  TenantVerifierV2,
+} from "../index.js";
+import {
+  Tenant,
+  TenantAttribute,
+  TenantFeature,
+  TenantKind,
+  TenantMail,
+  TenantMailKind,
+  TenantRevoker,
+  TenantUnitType,
+  TenantVerifier,
+  tenantAttributeType,
+  tenantKind,
+  tenantMailKind,
+  tenantUnitType,
+} from "./tenant.js";
+
+export function toFeatureV2(feature: TenantFeature): TenantFeatureV2 {
+  return match<TenantFeature, TenantFeatureV2>(feature)
+    .with({ type: "PersistentCertifier" }, (feature) => ({
+      sealedValue: {
+        oneofKind: "certifier",
+        certifier: {
+          certifierId: feature.certifierId,
+        },
+      },
+    }))
+    .exhaustive();
+}
+
+export function toTenantVerifierV2(verifier: TenantVerifier): TenantVerifierV2 {
+  return {
+    id: verifier.id,
+    verificationDate: BigInt(verifier.verificationDate.getTime()),
+    expirationDate: verifier.expirationDate
+      ? BigInt(verifier.expirationDate?.getTime())
```

Comment:
`verifier.expirationDate` has already been checked
```suggestion
      ? BigInt(verifier.expirationDate.getTime())
```

Reply:
Updated